easybuild/easyconfigs/m/MITgcmutils/MITgcmutils-0.1.2-foss-2022a.eb

```diff
@@ -0,0 +1,46 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'MITgcmutils'
+version = '0.1.2'
+
+homepage = 'https://mitgcm.org/'
+description = """A numerical model designed for study of the atmosphere, ocean, 
+and climate, MITgcm’s flexible non-hydrostatic formulation enables it to 
+efficiently simulate fluid phenomena over a wide range of scales; its adjoint 
+capabilities enable it to be applied to sensitivity questions and to parameter 
+and state estimation problems. By employing fluid equation isomorphisms, a 
+single dynamical kernel can be used to simulate flow of both the atmosphere 
+and ocean. The model is developed to perform efficiently on a wide variety of 
+computational platforms."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
```

Comment:
I do not see any dependency on tqdm, is this really needed?

Reply:
You are correct. Apologies for that, I have removed the surplus `tqdm`